pkg/termination/write_test.go

```diff
@@ -66,3 +66,29 @@ func TestExistingFile(t *testing.T) {
 		}
 	}
 }
+
+func TestMaxSizeFile(t *testing.T) {
+	value := ""
+	for n := 0; n < 4096; n++ {
```

Comment:
optionally `value := strings.Repeat("a", 4096)` could be used in place of the loop here.

Reply:
Great, thansk @sbwsg